.travis.php

```diff
@@ -19,8 +19,11 @@
     }
     echo "$dir\n";
 
-    $json = file_get_contents($dir.'/composer.json');
-    $package = json_decode($json);
+    $json = ltrim(file_get_contents($dir.'/composer.json'));
```

Comment:
the first `{` is still removed here: https://github.com/symfony/symfony/pull/16341/files#diff-d3241953fcadf5f8bc91210b1c064c2aR32